File: test/fixtures/output/node/request/compression.js
const request = require('request');

const options = {
  method: 'GET',
  url: 'http://mockbin.com/har',
  headers: {'accept-encoding': 'deflate, gzip, br'}
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: test/fixtures/output/java/nethttp/compression.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://mockbin.com/har"))
    .header("accept-encoding", "deflate, gzip, br")
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());